src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,15 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+    private final ColorSupplier cs = new ColorSupplier();
+
+    public Ball getRandomBall() {
+
+        Random random = new Random();
+        int value = random.nextInt(100);
+
+        return new Ball(cs.getRandomColor(), value);
```

Comment:
```suggestion
        return new Ball(cs.getRandomColor(), random.nextInt(UPPER_BOUND));
```
also lets add constants

Reply:
Done.